master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,69 @@
+---
+title: IP Pool Manipulation
+---
+The 
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+that Calico starts up with can be manipulated and changed.  This document
```

Comment:
Maybe consolidate this into one sentence with "This document describes how to make common modifications to Calico's IP Pools."